src/registry/mod.rs

```diff
@@ -177,19 +170,11 @@ pub struct PolicyArgs {
     pub display_policy_coverage: bool,
 }
 
-impl PolicyArgs {
-    /// Create a new empty `PolicyArgs` with the skip flag set to true.
-    pub fn skip() -> Self {
-        Self {
-            policies: Vec::new(),
-            skip_policies: true,
-            display_policy_coverage: false,
-        }
-    }
-}
-
 /// Manage a semantic convention registry and return the exit code.
-pub fn semconv_registry(log: impl Logger + Sync + Clone, command: &RegistryCommand) -> CmdResult {
+pub fn semconv_registry(
+    log: impl Logger + Sync + Clone + 'static,
```

Comment:
I didn’t understand why there was a need for a `static` lifetime for the logger. The logger is intended to be cloned (with low overhead) when there’s a need to pass it as a parameter to multiple sub-functions or methods.

Reply:
I've changed the design to avoid this now. I needed to own a struct that needed a logger. I figured adding the 'static lifetime would be ok since the underlying paris logger is 'static. But, anyway, I was able to move things around to avoid this.